wizard/templates/review-submission-template.php

```diff
@@ -0,0 +1,156 @@
+			<div class="instant-articles-card">
+				<div class="instant-articles-card-title">
+					<h3>Create More Articles to Submit for Review</h3>
+				</div>
+				<div class="instant-articles-card-content">
+					<div class="instant-articles-card-content-box instant-articles-card-content-full">
+						<p>
+							In order to begin publishing Instant Articles, our team needs to review a sample batch of <?php echo esc_html( Instant_Articles_Wizard_Review_Submission::MIN_ARTICLES ); ?>
+							of your Instant Articles. It looks like you don't have <?php echo esc_html( Instant_Articles_Wizard_Review_Submission::MIN_ARTICLES ); ?> articles available yet.
+						</p>
+						<p>
+							Once you've created the additional articles and have <?php echo esc_html( Instant_Articles_Wizard_Review_Submission::MIN_ARTICLES ); ?> available to send,
+							please return to this page and click the 'Submit for Review" button below.
+						</p>
+						<button id="instant-articles-wizard-submit-for-review" class="instant-articles-button-disabled instant-articles-button instant-articles-button-highlight">
+							<label>Submit for Review</label>
+						</button>
+					</div>
+				</div>
+			</div>
+		<?php endif; ?>
+
+
+	<?php break; ?>
+
+	<?php case Instant_Articles_Wizard_Review_Submission::STATUS_APPROVED : ?>
+		<div class="instant-articles-card instant-articles-card-success">
+			<div class="instant-articles-card-title">
+				<h3>Review Complete: Begin Publishing Your Instant Articles</h3>
+			</div>
+			<div class="instant-articles-card-content">
+				<div class="instant-articles-card-content-box instant-articles-card-content-full">
+					<p>
+						We reviewed some of your articles and they look great.
+						You're now ready to begin publishing your Instant Articles to Facebook and sharing them with your audience.
+					</p>
+					<p>
+						Next, set up monetization and analytics in Advanced Settings. Or explore the
+						<a href="https://www.facebook.com/<?php echo esc_attr( $fb_page_settings['page_id'] ); ?>/publishing_tools/?section=INSTANT_ARTICLES_SETTINGS" target="_blank">Instant Articles publishing tools</a>
+						through your selected Facebook Page.
+					</p>
+				</div>
+			</div>
+		</div>
+	<?php break; ?>
+
+	<?php case Instant_Articles_Wizard_Review_Submission::STATUS_REJECTED : ?>
+		<div class="instant-articles-card instant-articles-card-fail">
+			<div class="instant-articles-card-title">
+				<h3>Review Complete: Updates Needed</h3>
+			</div>
+			<div class="instant-articles-card-content">
+				<div class="instant-articles-card-content-box instant-articles-card-content-full">
+					<p>
+						We reviewed some of your articles and found a few things that need to be updated.
+					</p>
+					<p>
+						Please go to your selected
+						<a href="https://www.facebook.com/<?php echo esc_attr( $fb_page_settings['page_id'] ); ?>/publishing_tools/?section=INSTANT_ARTICLES_SETTINGS#Step-2" target="_blank">Facebook Page's Publishing Tools</a>
```

Comment:
`esc_url()` should be used for the whole URL here over `esc_attr()` for the variable